firefox-ios/firefox-ios-tests/Tests/XCUITests/PhotonActionSheetTests.swift

```diff
@@ -135,6 +155,14 @@ class PhotonActionSheetTests: FeatureFlaggedTestBase {
         mozWaitForElementToExist(app.staticTexts["Send to Device"])
     }
 
+    // https://mozilla.testrail.io/index.php?/cases/view/2306841
+    // Smoketest
+    func testSharePageWithShareSheetOptions_TAE() {
+        app.launch()
```

Comment:
Do we need app launch here? it is called right when entering the next method openNewShareShateet_TAE.. we may need just one?